controllers/utils/utils.go

```diff
@@ -40,3 +43,44 @@ func ErrFunc(err error) controllerutil.MutateFn {
 		return err
 	}
 }
+
+// IPFSContainerResources Returns the resource requests/requirements for running a single IPFS Container
+// depending on the storage requested by the user.
+func IPFSContainerResources(ipfsStorageBytes int64) (ipfsResources corev1.ResourceRequirements) {
```

Comment:
@coryschwartz I changed these values around because the GitHub actions runner wouldn't allow more than 1 replica to run in the e2e-tests at a time, leading them to fail.